src/team/crazynetwork/raids/skyfactory/Crook.java

```diff
@@ -0,0 +1,84 @@
+        crookMeta.setDisplayName(ChatColor.RESET + "" + ChatColor.WHITE + "Crook " + ChatColor.DARK_GREEN + "64/64"); //Display name as: Crook 64/64
+        crookMeta.setLore(Arrays.asList(ChatColor.RESET + "" + ChatColor.GRAY + "" + ChatColor.ITALIC + "Increases chance of getting a sapling drop", ChatColor.RESET + "" + ChatColor.DARK_GRAY + "64/64", ChatColor.RESET + "" + ChatColor.LIGHT_PURPLE + "" + ChatColor.MAGIC + "OBFUSCATED"));
+        crookItem.setItemMeta(crookMeta); //Setting the name and lore
+        ShapedRecipe crook1 = new ShapedRecipe(crookItem).shape("ss ", " s ", " s ").setIngredient('s', Material.STICK); //Initialing recipe
+        ShapedRecipe crook2 = new ShapedRecipe(crookItem).shape(" ss", " s ", " s ").setIngredient('s', Material.STICK); //Initialing alternate recipe
+        ShapedRecipe[] recipes = {crook1, crook2};
+        return recipes;
+    }
+
+    @EventHandler
+    public void onBlockBreak(BlockBreakEvent e) {
+        if (e.getBlock().getType() == Material.LEAVES || e.getBlock().getType() == Material.LEAVES_2) {
+            //The Block mined is a leaf block
+            try {
+                if (e.getPlayer().getItemInHand().getType() == Material.STICK)
+                    if (e.getPlayer().getItemInHand().hasItemMeta())
+                        if (e.getPlayer().getItemInHand().getItemMeta().getDisplayName().contains(" "))
+                            if (ChatColor.stripColor(e.getPlayer().getItemInHand().getItemMeta().getDisplayName()).split(" ")[0].equalsIgnoreCase("Crook"))
+                                if (e.getPlayer().getItemInHand().getItemMeta().hasLore())
+                                    if (e.getPlayer().getItemInHand().getItemMeta().getLore().size() >= 2)
+                                        if (ChatColor.stripColor(e.getPlayer().getItemInHand().getItemMeta().getLore().get(0)).equalsIgnoreCase("Increases chance of getting a sapling drop")) {
+                                            //The item is most held likely a crook
+                                            if (ChatColor.stripColor(e.getPlayer().getItemInHand().getItemMeta().getLore().get(1)).equalsIgnoreCase("1/64")) {
+                                                //The crook is out of durability now
+                                                if (e.getPlayer().getItemInHand().getAmount() == 1) {
+                                                    e.getPlayer().setItemInHand(null); //If there is only 1 crook in their hand, just remove everything from the hand
+                                                } else {
+                                                    ItemStack c = recipe()[0].getResult();
+                                                    c.setAmount(e.getPlayer().getItemInHand().getAmount() - 1);
+                                                    e.getPlayer().setItemInHand(c); //If there is more than 1 crook, delete 1 of them and make the rest full durability
```

Comment:
Maybe times two the durability? If a player decide to stack two durability 1 crooks and break a leaf, he will get a full durability crook. Only items with same lore stack.